lib/tests/streamlit/elements/element_utils_test.py

```diff
@@ -107,6 +108,8 @@ def test_check_session_state_rules_prints_warning(
         mock_session_state = MagicMock()
         mock_session_state.is_new_state_value.return_value = True
         patched_get_session_state.return_value = mock_session_state
+        # Reset globale flag:
+        utils._shown_default_value_warning = False
```

Comment:
This seems to be needed since otherwise the flag will already have been set to True by the date_input_test.py